libs/portals/admin/signature-collection/src/module.tsx

```diff
@@ -0,0 +1,33 @@
+import { PortalModule } from '@island.is/portals/core'
+import { lazy } from 'react'
+import { m } from './lib/messages'
+import { SignatureCollectionPaths } from './lib/paths'
+import { listsLoader } from './screens/AllLists/AllLists.loader'
+import { AdminPortalScope } from '@island.is/auth/scopes'
+import { listLoader } from './screens/List/List.loader'
+
+const AllLists = lazy(() => import('./screens/AllLists'))
+const List = lazy(() => import('./screens/List'))
+
+const allowedScopes: string[] = [AdminPortalScope.petitionsAdmin]
```

Comment:
As this is a standalone admin module we should use a new separate scope for access control. And looks like you still have to look into scopes on the gql domain.